pkg/version/version.go

```diff
@@ -38,7 +38,7 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                           = "0.7.0-beta.1.dev.5"
+	Promscale                           = "0.7.0"
 	PrevReleaseVersion                  = "0.7.0-beta.1"
 	PromMigrator                        = "0.0.2"
```

Comment:
We need to bump the patch here as we do have a new commit for prom-migrator https://github.com/timescale/promscale/commit/b2f41d5da7481d8a779568bb778ae0dba7fbf557#diff-2eaa38bcaeeee666a97b66de9efff82d6b4bdf3dda7df693b9e1cf58940cc342

Reply:
We also have to bump jaeger-proxy because of https://github.com/timescale/promscale/commit/aaa3f83caf15c9717c4351f0142ca1625cc1ece6#diff-792b22d12d8ee4227140a940fbff83b8d78f52d9f025fe841241d5c266f4b9cc